.github/workflows/ci.yml

```diff
@@ -21,6 +21,17 @@ on:
     - cron: "0 8 * * *"
 
 jobs:
+  test-build-front-end:
```

Comment:
Wouldn't failures here be caught in the lint CI step? We can make it more explicit, but I think we are already covered by that step.

Reply:
Ah right, that's already over in the other file :+1:

Reply:
For clarity, we could move the build to here and leave the lint there. It's rather badly named at the moment.